unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/AvatarSlot/AvatarSlotComponentView.cs

```diff
@@ -49,6 +56,12 @@ public override void Awake()
                 unequipButton.gameObject.SetActive(false);
             });
             ResetSlot();
+            tooltipContainer.gameObject.SetActive(true);
+            tooltipDefaultPosition = new Vector2(30, 120);
+            tooltipFullPosition = new Vector2(30, 150);
+            tooltipContainer.anchoredPosition = tooltipDefaultPosition;
+            tooltipContainer.gameObject.SetActive(false);
+            nftContainerDefaultPosition = nftContainer.anchoredPosition;
```

Comment:
Can we encapsulate all these code lines into a method with some descriptive name that describes what we're doing?

Reply:
done